crates/stages/src/stages/senders.rs

```diff
@@ -28,6 +28,7 @@ pub struct SendersStage {
     pub commit_threshold: u64,
 }
 
+// TODO(onbjerg): Should unwind
```

Comment:
In which cases?

Reply:
All cases I think. If we can't recover the sender then the transaction is invalid (assuming our recover algo is not broken)